docs/en/setup/backend/backend-alarm.md

```diff
@@ -348,3 +348,23 @@ which will override the settings in `alarm-settings.yml`.
 In order to determine whether an alerting rule is triggered or not, SkyWalking needs to cache the metrics of a time window for
 each alerting rule. If any attribute (`metrics-name`, `op`, `threshold`, `period`, `count`, etc.) of a rule is changed,
 the sliding window will be destroyed and re-created, causing the Alarm of this specific rule to restart again.
+
+### Configuration property key of Alerting element
```

Comment:
What is this doc about? What property keys? Could you point to the source codes about this?

Reply:
[RulesReader](https://github.com/apache/skywalking/blob/master/oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RulesReader.java#L85)

There is no doc list these key

Reply:
I haven searched  these key in all MD doc.

Reply:
```suggestion
### Keys with data types of alerting rule configuration file
```